data/korath/korath hails.txt

```diff
@@ -0,0 +1,1112 @@
+		"ralefab nrag lemotakteb'"           # spectacularly exhibits [its purpose]
+		"'oleref-'olek ipihemef avlerit"     # perennially exceeds performance [metrics]
+		"kaohis ranres kanra' tiah"          # is most excellently fashioned
+
+phrase "exile: [system] is better than yours"
+	word
+		"saha-sahaleh ikacemef inumurtek asra"     # is centuries past your technology
+		"kapimlek inah asra iklebe' refolk'ri ipami"  # looks to you like a divine instrument
+		"raavleteb kad nrag pihem tiah ilas mabat" # [has a] better paint job than your ship
+		"monnro is laom rafapafeb asra"          # excels beyond your understanding
+		"pamasa ire' nrag tasik parleb a'ih asra ianlap" # is art you could never appreciate
+
+phrase "exile: Our ship is disabled."
+	phrase
+		"exile: We or our ship"
+	word
+		" "
+	phrase
+		"exile: ain't doing so well right now."
+
+
+phrase "exile: Our systems went haywire."
+	phrase
+		"exile: Our malfunctioning system"
+	word
+		" "
+	phrase
+		"exile: has destroyed our ship."
+
+phrase "exile: Our malfunctioning system"
+	phrase
+		"exile: Our ship's"
+	word
+		" "
+	phrase
+		"exile: semi-translated weapons"  12
+		"exile: semi-translated engines"  7
+		"exile: semi-translated systems"  7
+
+phrase "exile: Our ship's"
+	word
+		"Akit"                # Our
+		"Mabat iri"           # This ship's
+		"Ot"                  # My
+		"Oki"                 # The
+
+# These outfit names are just supposed to be good enough for hails.
+# If we were to rename the outfits, the names should be finely
+# crafted after wide-spread review.
+
+phrase "exile: semi-translated weapons"
+	word
+		"mihfa etilk' kellok"           # grab-strike turret
+		"alaref ratunufeb krop'"        # shunt-strike turret
+		"oamapnref alaref i'akiqaln"    # banisher grav-turret
+		"ratlaomet akavre' i'akiqaln"   # expeler grav-ray
+		"alaref matnrareb masol"        # warder anti-missile
+		"akavre' tahfuk iba"            # fire-lance
+		"nraseb nrag latahlek"          # blaze-pike turret
+		"alaref ataler"                 # [korath] inferno turret
+		"le'am ranrahfakleb"            # digger mining beam
+		"alaref ranrahfakleb"           # digger mining turret
+		"mebaket enralnam"              # langrage hyper-heaver
+		"masol iba ranril"              # firelight missile
+		"masol iba rabuk"               # firestorm torpedo
+		"lodromeb rabuk iba"            # firestorm battery
+
+
+phrase "exile: semi-translated engines"
+	word
+		"nrulusreb ovaf"                # forward thrusters
+		"nrulusreb timahlek"            # reverse thrusters
+		"latahfeb 'alet"                # afterburners
+		"kev isofet"                    # steering
+		"ri'ef"                         # engines
+		"talennreb lo'oh"               # bow drive
+
+		"tusters"                       # thrusters (mispronounced)
+		"injuns"                        # engines (mispronounced)
+		"tearing"                       # steering (mispronounced)
```

Comment:
I think "tearing" has too much of a different connotation to be used as a mispronunciation of steering.

Reply:
I'm changing it to "s'earing" in my next push.